data/hai/hai reveal 5 philosophers.txt

```diff
@@ -924,11 +940,27 @@ mission "Hai Reveal [B12] Devil-Hide Battle"
 			variant
 				"Palavret"
 
+
+fleet "Elenchus Tribute"
```

Comment:
This fleet is definitely more in line with the combat abilities the player has at the moment. On its own, it's unimpressive for a 5000 tribute colony, but they do have a steady supply of powerful warships to better back their claim.

Reply:
I dropped the 7k to 5k after discussing it with Grey who thinks its way too high anyway, but I'll leave proper tuning of how much (and if you keep the tribute, which you shouldnt in my opinion)

rn I just kept it in line with what the new capabilities are relative to the old ones

Reply:
Keeping some tribute makes sense from a story standpoint as a gratitude payment for having freed their people from slavery after 200 years.  For such a remote colony, 5000/day seems high. It should be tuned along with the battle and content though, and we need the new warlord ship to tune the battle any further.

Reply:
It should be explained in another PR though that that is what it is

Reply:
~~I got enough on my PR pile as is honestly and I dont feel like there are many ways to address it~~
I'll talk about it with Grey when they get the time